ansible/modules/hashivault/hashivault_auth_method.py

```diff
@@ -71,6 +71,7 @@
     config:
         description:
             - configuration set on auth method. expects a dict
```

Comment:
This document *seems* wrong to me like this should be a yaml dictionary rather than a string.

Reply:
you talking about the `default` line? yeah, interestingly it works either way. when i first submitted the PR it was in standard `{}` format but after i started getting the sphinx issues i was messing with the format because i didnt know what was borking it.

i agree though, the doc example should show without the `"`, though it does work either way